File: Src/App/zabotalaboratory_client/src/app/shared/forms/laboratory-analyses/add-laboratory-analyses.form.ts
import {AddTalonsForm} from "./add-talons.form";

export class AddLaboratoryAnalysesForm {
  public firstName: string;
  public lastName: string;
  public patronymicName: string;
  public dateOfBirth: string;
  public genderId: number;
  public clinicId: number;
  public doctor: string;
  public talons: AddTalonsForm[];
}

File: Src/App/zabotalaboratory/zabotalaboratory.Analyses.Forms/LaboratoryAnalyses/AddLaboratoryAnalysesForm.cs
using System.Collections.Generic;

namespace zabotalaboratory.Analyses.Forms.LaboratoryAnalyses
{
    public class AddLaboratoryAnalysesForm
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string PatronymicName { get; set; }

        public string DateOfBirth { get; set; }

        public int GenderId { get; set; }

        public int ClinicId { get; set; }

        public string Doctor { get; set; }

        public IEnumerable<AddTalonsForm> Talons { get; set; }

    }
}
